tests/behat/src/FixtureContext.php

```diff
@@ -118,22 +118,6 @@ public function iSeeTheRadioButtonAttributeEquals($radioLabel, $attribute, $valu
         Assert::assertEquals($value, $radioButton->getAttribute($attribute));
     }
 
-    /**
-     * Assumes you've just opened the Insert link menu, e.g.
-     * I click on the "div[aria-label='Insert link [Ctrl+K]'] button" element
-     *
-     * @When /^I select "(.+?)" from the TinyMCE menu with javascript$/
-     * @param string $label
-     */
-    public function iSelectFromTheTinyMceMenu($label)
```

Comment:
This is no longer used - it was only used in the above tests where it is no longer necessary.